### DISCUSSION TITLE
Minimum example of making a table of content using `@iles/heading`

### DISCUSSION THREAD

**Discussion by kecrily** [ElMassimo/iles#96] [Q&A] [answered]
I would like to use `@iles/heading` to create a table of content, but the documentation is too brief, and looking at the source code directly from the documentation site is too cumbersome.

Is it possible to provide minimal examples of how to implement table of content?

**Comment by ElMassimo** [comment, marked as answer]
You can use the [`TableOfContents`](https://github.com/ElMassimo/iles/blob/main/docs/src/components/TableOfContents.vue#L12) component as a starting point.

The `meta.headings` property is an array of [`{ level, title, slug }`](https://github.com/ElMassimo/iles/blob/main/packages/headings/src/headings.ts#L13-L17), and can be used in different ways.

---

I intend to make a recipes/cookbook section in the docs where it covers common use cases, but I don't have a timeline for it.
